.github/workflows/pr-project-label.yml

```diff
@@ -9,6 +9,10 @@ jobs:
   label_project:
     runs-on: ubuntu-latest
     steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.7.0
+      with:
+          access_token: ${{ github.token }}
```

Comment:
Super nitpick: maybe add a space here to conform with other files

Reply:
Thanks! Updated as suggested. 9a4138846d61a135fa91befbad19aff3b353b864